src/targets/gpu/compile_miopen.cpp

```diff
@@ -29,7 +29,9 @@
 #include <migraphx/make_op.hpp>
 #include <migraphx/register_op.hpp>
 #include <migraphx/op/identity.hpp>
-#include <migraphx/gpu/rocblas.hpp>
+#ifdef MIGRAPHX_USE_ROCBLAS
```

Comment:
This should be defined as 1 or 0, so it should use `#if MIGRAPHX_USE_ROCBLAS`. Also I dont see you adding this define to cmake.